RecoEgamma/EgammaElectronProducers/python/gedGsfElectronFinalizer_cfi.py

```diff
@@ -1,12 +1,16 @@
 import FWCore.ParameterSet.Config as cms
 
-gedGsfElectrons = cms.EDProducer("GEDGsfElectronFinalizer",
-                                 previousGsfElectronsTag = cms.InputTag("gedGsfElectronsTmp"),
-                                 pfCandidatesTag = cms.InputTag("particleFlowTmp"),
-                                 pfIsolationValues = cms.PSet(
-                                       pfSumChargedHadronPt = cms.InputTag('gedElPFIsoValueCharged03'),
-                                       pfSumPhotonEt = cms.InputTag('gedElPFIsoValueGamma03'),
-                                       pfSumNeutralHadronEt= cms.InputTag('gedElPFIsoValueNeutral03'),
-                                       pfSumPUPt = cms.InputTag('gedElPFIsoValuePU03')),
-                                 outputCollectionLabel = cms.string("")
-                                 )
+from RecoEgamma.EgammaTools.regressionModifier_cfi import *
+
+gedGsfElectrons = cms.EDProducer(
+    "GEDGsfElectronFinalizer",
+    previousGsfElectronsTag = cms.InputTag("gedGsfElectronsTmp"),
+    pfCandidatesTag = cms.InputTag("particleFlowTmp"),
+    regressionConfig = regressionModifier.clone(rhoCollection=cms.InputTag("fixedGridRhoFastjetAllTmp")),
```

Comment:
it may have been somewhat nicer to just have this extra line show up in diffs, instead of everything else as well due to reindentation